src/routes/props/Badge.json

```diff
@@ -1 +1 @@
-{"props":[["color","'blue' | 'dark' | 'red' | 'green' | 'yellow' | 'indigo' | 'purple' | 'pink' ","'blue'"],["large","boolean ","false"],["href","string | undefined ","undefined"],["rounded","boolean ","false"],["index","boolean ","false"],["dismissable","boolean ","false"],["baseClass","string ","'inline-flex items-center justify-center -mb-0.5'"]]}
\ No newline at end of file
+{"props":[["color","'blue' | 'dark' | 'red' | 'green' | 'yellow' | 'indigo' | 'purple' | 'pink' ","'blue'"],["large","boolean ","false"],["border","boolean ","false"],["href","string | undefined ","undefined"],["rounded","boolean ","false"],["index","boolean ","false"],["dismissable","boolean ","false"],["closeBtnSize","'xs' | 'sm' | 'md' ","'sm'"]]}
```

Comment:
Re-run `npm run gen:props` as `closeBtnSize` is removed in the latest version, isn't it?